dipy/direction/peaks.py

```diff
@@ -661,6 +672,7 @@ def reshape_peaks_for_visualization(peaks):
     return peaks.reshape(np.append(peaks.shape[:-2], -1)).astype("float32")
 
 
+@warning_for_keywords()
```

Comment:
not needed since `*` was there from the beginning

Reply:
you need to remove the decorator in the line above. This function always existed with * so we do not need to warn the user